aodntools/timeseries_products/Documentation/Hourly_timeseries.md

```diff
@@ -0,0 +1,157 @@
+`DOX2` (moles_of_oxygen_per_unit_mass_in_sea_water);  
+ `DOXS` (fractional_saturation_of_oxygen_in_sea_water);  
+ `PAR` (downwelling_photosynthetic_photon_flux_in_sea_water).  
+  
+
+
+
+Each of the VoIs are produced by selecting only the “good” and “probably good” data (QC flags 1 and 2), binning the values to a one-hour interval, and concatenating the resulting values into a single file. The resulting variables have dimension OBSERVATION. The VoI’s ancillary_variables, in particular the corresponding quality-control flags, are not included. 
+
+An additional version of the product will include “good”, “probably good” and “non QC performed” data. In this case and for each variable, the percentage of QCed values will be indicated as a variable attribute.
+
+The binning intervals will be one hour long, centred on the hour (i.e. HH:00:00). The binning method will depend on the type of variable being:
+
+| Variables | Method | 
+|-----------|--------|
+| TEMP, CNDC, PRES, PRES_REL,  DEPTH, PSAL, DOXY, DOX, DOX1, DOX1_1, DOX1_2, DOX1_3, DOX2, DOX2_1, DOXS | Mean | 
+| FLU2, CPHL, CHLU, CHLF, TURB, TURBF, PAR | Median | 
+
+The binned `TIME` values are concatenated into a variable `TIME(OBSERVATION)`.  As input files can have overlapping or non-contiguous time coverage, the TIME variable may have duplicate values or gaps.
+
+The binned `DEPTH` values from input files are concatenated into a variable `DEPTH(OBSERVATION)`. If not present, fill values are stored. The variables `PRES` (sea_water_pressure) and `PRES_REL` (sea_water_pressure_due_to_sea_water) will be aggregated in exactly the same way as `DEPTH`. Every aggregated file will contain `DEPTH`, `PRES`, and `PRES_REL`.
+
+Additional metadata variables (ancillary variables) will be included, derived from the binning process and the source files: 
+`VoI_std`: standard deviation per averaging bin;
+`VoI_min`: minimum value per bin;
+`VoI_max`: maximum value per bin;
+`VoI_count`: number of observations in bin;
+
+In order to keep track of the provenance of VoIs in the aggregated file, accessory variables are created:
+
+- `instrument_index(OBSERVATION)`: index [0:number of files] of the instrument used, referencing the INSTRUMENT dimension.
+- `source_file(INSTRUMENT, string256)`: URLs of the files used
+- `instrument_id(INSTRUMENT, string256)`: concatenated deployment_code, instrument and instrument_serial_number from the global attributes of each file
+- `LATITUDE(INSTRUMENT)`: LATITUDE per instrument.
+- `LONGITUDE(INSTRUMENT)`: LONGITUDE per instrument.
+- `NOMINAL_DEPTH(INSTRUMENT)`: nominal depth per instrument, from the input file’s variable NOMINAL_DEPTH or global attribute instrument_nominal_depth.
+
+The global metadata will be a set of IMOS standard attributes, and specific standard attributes concatenated from input file attributes
+
+Product files will be made available via the AODN THREDDS server, and regularly updated to include new data.
```

Comment:
Remove the last two sentences. The one about global metadata is included below. The second one is about the project, not the code.